From 6d79ba8802f5c3357362b9a2be49f7ca29246235 Mon Sep 17 00:00:00 2001
From: Igal Zeltser <izeltser@habana.ai>
Date: Tue, 27 Aug 2024 17:33:37 +0300
Subject: [PATCH] [FSW-8566] fs1: fix misc arc address

MISC ARC is implemented in IO die HALF1, address is adjusted
respectively

Change-Id: Id3025c7fb6ce82dba8042b7c5cbb3d25b116c6bd
Signed-off-by: Igal Zeltser <izeltser@habana.ai>
---

diff --git a/arch/fs1/arc/common.h b/arch/fs1/arc/common.h
index 338db3c..17ee764 100644
--- a/arch/fs1/arc/common.h
+++ b/arch/fs1/arc/common.h
@@ -37,9 +37,9 @@
 #define ARC_PID_ICCM_SIZE		SZ_128K
 #define ARC_PID_DCCM_FULL_BASE		0x300007FF88E0000ull
 #define ARC_PID_DCCM_SIZE		SZ_64K
-#define ARC_MISC_ICCM_FULL_BASE		0x300007FF8900000ull
+#define ARC_MISC_ICCM_FULL_BASE		0x300007FFA900000ull
 #define ARC_MISC_ICCM_SIZE		SZ_256K
-#define ARC_MISC_DCCM_FULL_BASE		0x300007FF8940000ull
+#define ARC_MISC_DCCM_FULL_BASE		0x300007FFA940000ull
 #define ARC_MISC_DCCM_SIZE		SZ_64K
 
 /* CFG is at Arc region F, specs CFG base address is 0x70000000. 7+8 = 0xF */
